device/dell/x86_64-dellemc_z9332f_d1508-r0/DellEMC-Z9332f-M-O16C64/sai_postinit_cmd.soc

```diff
@@ -0,0 +1,2530 @@
+delay 200
+link off
+counter off
+
+#***
+#*** Port CD0 Preemphasis setting ***
+#***
+
+local port cd0
+#*** lane 0 ***
+phy $port TXFIR_TAP_CTL0r.0 TXFIR_TAP0_COEFF=4
```

Comment:
why are we having this? can we configure it via xrcvd?

Reply:
The 6tap settings needed for Tomahawk3 lane settings are not currently supported in xcvrd. We are currently pursuing this support with Broadcom and the SAI implementation (I am still confirming this though), but for the time being we have to define the 6tap settings statically here.

Once the 6tap settings are enabled in SAI, we can remove this and instead define these values in `media_settings.json` for the Z9332.